lib/v2/screens/sign_up/create_username/create_username.dart

```diff
@@ -82,18 +99,40 @@ class _CreateUsernameState extends State<CreateUsername> {
     );
   }
 
+  String? _validateUsername(String? username) {
```

Comment:
This needs to live in the state mapper. 

1 - Fire an event when the text changes. 
2 - Handle the event in the bloc/state mapper. 
3 - update the state accordingly.

Reply:
Addressed